### DISCUSSION TITLE
CluserAdmin override of Non-Admin Backup CR settings e.g TTL

### DISCUSSION THREAD

**Discussion by weshayutin** [migtools/oadp-non-admin#33] [General]
Cluster Admins should be able to set defaults and or limits on specific settings of a non-admin backup cr.  An example would be the TTL time to live setting on a backup.  Cluster Admins could limit the TTL a non-admin user can set to save space in remote or local storage.

* If this feature is important to you please let us know.

**Comment by mateusoliveira43** [comment]
resolved by https://github.com/migtools/oadp-non-admin/pull/110
